config/class-site-details-index.php

```diff
@@ -228,6 +252,100 @@ public function get_jetpack_info() {
 		return $jetpack_info;
 	}
 
+	/**
+	 * Gather all the information about Parsely
+	 */
+	public function get_parsely_info() {
+		$parsely      = get_option( 'parsely' );
+		$parsely_info = [];
+
+		$parsely_info['active']           = false;
+		$parsely_info['integration_type'] = $this->get_parsely_integration_type();
+
+		if ( self::PARSELY_INTEGRATION_TYPE_MUPLUGINS === $parsely_info['integration_type'] ||
+			self::PARSELY_INTEGRATION_TYPE_MUPLUGINS_SILENT === $parsely_info['integration_type'] ||
+			self::PARSELY_INTEGRATION_TYPE_SELF_MANAGED === $parsely_info['integration_type']
+		) {
+			$parsely_info['active'] = true;
+		}
+
+		$parsely_info['parsely_service_type'] = $this->get_parsely_service_type( $parsely );
+		$parsely_info['version']              = $this->get_parsely_version( $parsely );
+
+		return $parsely_info;
+	}
+
+	/**
+	 * Gets the Parsely version
+	 * @param array $parsely The Parsely options.
+	 * @return string The version.
+	 */
+	public function get_parsely_version( $parsely ) {
+		if ( ! $parsely || ! isset( $parsely['plugin_version'] ) ) {
+			$supported_versions = \Automattic\VIP\WP_Parsely_Integration\SUPPORTED_VERSIONS;
+			$plugin_version     = apply_filters( 'wpvip_parsely_version', $supported_versions[0] );
+
+			return $plugin_version;
+		}
+
+		return $parsely['plugin_version'];
+	}
+
+	/**
+	 * The way that the plugin is used in WordPress
+	 */
+	public function get_parsely_integration_type() {
+		// is enabled through mu-plugins
+		if ( has_filter( 'wpvip_parsely_load_mu' ) ) {
+			if ( has_filter( 'wpvip_parsely_hide_ui_for_mu' ) ) {
+				return self::PARSELY_INTEGRATION_TYPE_MUPLUGINS_SILENT;
+			}
+
+			return self::PARSELY_INTEGRATION_TYPE_MUPLUGINS;
+		}
+
+		// is enabled as a standard plugin
+		$plugin = 'wp-parsely/wp-parsely.php';
+		if ( in_array( $plugin, (array) get_option( 'active_plugins', array() ), true ) || is_plugin_active_for_network( $plugin ) ) {
+			return self::PARSELY_INTEGRATION_TYPE_SELF_MANAGED;
+		};
+
+		return self::PARSELY_INTEGRATION_TYPE_NONE;
+	}
+
+	/**
+	 * The way that the service is being used
+	 * @param array $parsely The Parsely options.
+	 * @return string service type.
+	 */
+	public function get_parsely_service_type( $parsely ) {
+		if ( is_array( $parsely ) && isset( $parsely['apikey'] ) && '' !== $parsely['apikey'] ) {
+			$result = $this->call_parsely_api( $parsely );
```

Comment:
Leaving for historical purposes — there were concerns about this API call flagged on Slack by @WPprodigy